From 3b8dd4c23cc1f16b4279f367a67848a3362e22d5 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:10:44 -0400
Subject: [ethernet] i40e: Fix device ID define names to align to standard

Message-id: <1409663591-32531-2-git-send-email-sassmann@redhat.com>
Patchwork-id: 90478
O-Subject: [RHEL7.1 PATCH 001/148] i40e: Fix device ID define names to align to standard
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Shannon Nelson <shannon.nelson@intel.com>

Rework the device ID #defines to follow the _DEV_ID convention
already established in the other Intel drivers.

Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Shannon Nelson <shannon.nelson@intel.com>
Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit ab60085e2e6e48ad4dc3d25254239e1119d074a4)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_common.c b/drivers/net/ethernet/intel/i40evf/i40e_common.c
index 44b90b3..7b13953 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_common.c
+++ b/drivers/net/ethernet/intel/i40evf/i40e_common.c
@@ -39,20 +39,20 @@ i40e_status i40e_set_mac_type(struct i40e_hw *hw)
 
  if (hw->vendor_id == PCI_VENDOR_ID_INTEL) {
   switch (hw->device_id) {
-  case I40E_SFP_XL710_DEVICE_ID:
-  case I40E_SFP_X710_DEVICE_ID:
-  case I40E_QEMU_DEVICE_ID:
-  case I40E_KX_A_DEVICE_ID:
-  case I40E_KX_B_DEVICE_ID:
-  case I40E_KX_C_DEVICE_ID:
-  case I40E_KX_D_DEVICE_ID:
-  case I40E_QSFP_A_DEVICE_ID:
-  case I40E_QSFP_B_DEVICE_ID:
-  case I40E_QSFP_C_DEVICE_ID:
+  case I40E_DEV_ID_SFP_XL710:
+  case I40E_DEV_ID_SFP_X710:
+  case I40E_DEV_ID_QEMU:
+  case I40E_DEV_ID_KX_A:
+  case I40E_DEV_ID_KX_B:
+  case I40E_DEV_ID_KX_C:
+  case I40E_DEV_ID_KX_D:
+  case I40E_DEV_ID_QSFP_A:
+  case I40E_DEV_ID_QSFP_B:
+  case I40E_DEV_ID_QSFP_C:
    hw->mac.type = I40E_MAC_XL710;
    break;
-  case I40E_VF_DEVICE_ID:
-  case I40E_VF_HV_DEVICE_ID:
+  case I40E_DEV_ID_VF:
+  case I40E_DEV_ID_VF_HV:
    hw->mac.type = I40E_MAC_VF;
    break;
   default:
diff --git a/drivers/net/ethernet/intel/i40evf/i40e_type.h b/drivers/net/ethernet/intel/i40evf/i40e_type.h
index beb3fa7..3bffac0 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -32,22 +32,22 @@
 #include "i40e_lan_hmc.h"
 
 /* Device IDs */
-#define I40E_SFP_XL710_DEVICE_ID 0x1572
-#define I40E_SFP_X710_DEVICE_ID  0x1573
-#define I40E_QEMU_DEVICE_ID  0x1574
-#define I40E_KX_A_DEVICE_ID  0x157F
-#define I40E_KX_B_DEVICE_ID  0x1580
-#define I40E_KX_C_DEVICE_ID  0x1581
-#define I40E_KX_D_DEVICE_ID  0x1582
-#define I40E_QSFP_A_DEVICE_ID  0x1583
-#define I40E_QSFP_B_DEVICE_ID  0x1584
-#define I40E_QSFP_C_DEVICE_ID  0x1585
-#define I40E_VF_DEVICE_ID  0x154C
-#define I40E_VF_HV_DEVICE_ID  0x1571
-
-#define i40e_is_40G_device(d)  ((d) == I40E_QSFP_A_DEVICE_ID  || \
-      (d) == I40E_QSFP_B_DEVICE_ID  || \
-      (d) == I40E_QSFP_C_DEVICE_ID)
+#define I40E_DEV_ID_SFP_XL710 0x1572
+#define I40E_DEV_ID_SFP_X710  0x1573
+#define I40E_DEV_ID_QEMU  0x1574
+#define I40E_DEV_ID_KX_A  0x157F
+#define I40E_DEV_ID_KX_B  0x1580
+#define I40E_DEV_ID_KX_C  0x1581
+#define I40E_DEV_ID_KX_D  0x1582
+#define I40E_DEV_ID_QSFP_A  0x1583
+#define I40E_DEV_ID_QSFP_B  0x1584
+#define I40E_DEV_ID_QSFP_C  0x1585
+#define I40E_DEV_ID_VF  0x154C
+#define I40E_DEV_ID_VF_HV  0x1571
+
+#define i40e_is_40G_device(d)  ((d) == I40E_DEV_ID_QSFP_A  || \
+      (d) == I40E_DEV_ID_QSFP_B  || \
+      (d) == I40E_DEV_ID_QSFP_C)
 
 #define I40E_MAX_VSI_QP   16
 #define I40E_MAX_VF_VSI   3
diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 06bf825..f5caf44 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -45,7 +45,7 @@ static const char i40evf_copyright[] =
  *   Class, Class Mask, private data (not used) }
  */
 static DEFINE_PCI_DEVICE_TABLE(i40evf_pci_tbl) = {
- {PCI_VDEVICE(INTEL, I40E_VF_DEVICE_ID), 0},
+ {PCI_VDEVICE(INTEL, I40E_DEV_ID_VF), 0},
  /* required last entry */
  {0, }
 };
-- 
1.7.1